org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/util/JsonUtilTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2019 Sreekanth Nelakurthi
```

Comment:
Sorry, the license should be an exact copy of the [LICENSE_header](https://github.com/aposin/LicenseScout/blob/master/org.aposin.licensescout.core/LICENSE_header.txt). Please, fix it to be able to merge.

Reply:
Done.

Reply:
Thanks and sorry for the wrong indication. I will add my suggestion as a commit to avoid another round of review from your side.